portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,81 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.flex-container {
+	display: flex;
+}
+
+/* Intro Section */
+#profile-pic {
+	border-radius: 50%;
+  width: 100px;
+  margin: 10px 20px;
```

Comment:
keep the properties in alphabetical order. that makes it easy to find properties within a css rule.

border-radius
margin
width